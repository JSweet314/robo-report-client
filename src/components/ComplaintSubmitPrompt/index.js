import React from 'react';

const ComplaintSubmitPrompt = () => {
  return (
    <div className='complaint-submit-prompt'>
      <p>
        Once you hit the <strong>Report It!</strong> button, the FCC form will pull up
        and your answers will appear. Just remember,
      </p>
      <ol>
        <li>
        Scoll through and review the form and make any edits you&#39;d
        like.
        </li>
        <li>Check the Captcha that you are not a robot (ironic, we know)</li>
        <li>Submit the form!</li>
      </ol>
      <p>
        Once finished, you should receive an email from the FCC confirming your 
        report. Check out the <strong>My Reports</strong> section to view past reports.
      </p>
    </div>
  );
};

export default ComplaintSubmitPrompt;