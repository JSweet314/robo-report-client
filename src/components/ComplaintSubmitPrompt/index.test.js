import React from 'react';
import { shallow } from 'enzyme';
import ComplaintSubmitPrompt from './index';

describe('ComplaintSubmitPrompt', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ComplaintSubmitPrompt />);
  });

  it('should match a snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});