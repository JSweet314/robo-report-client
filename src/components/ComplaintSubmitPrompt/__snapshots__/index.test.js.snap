// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComplaintSubmitPrompt should match a snapshot 1`] = `
<div
  className="complaint-submit-prompt"
>
  <p>
    Once you hit the 
    <strong>
      Report It!
    </strong>
     button, the FCC form will pull up and your answers will appear. Just remember,
  </p>
  <ol>
    <li>
      Scoll through and review the form and make any edits you'd like.
    </li>
    <li>
      Check the Captcha that you are not a robot (ironic, we know)
    </li>
    <li>
      Submit the form!
    </li>
  </ol>
  <p>
    Once finished, you should receive an email from the FCC confirming your report. Check out the 
    <strong>
      My Reports
    </strong>
     section to view past reports.
  </p>
</div>
`;
